.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 21
+  Max: 25
```

Comment:
- [ ] I would not recommend editing the `rubocop.yml` file. Please use the file without editing or updating. You can find this [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml).